src/js/node/tls.ts

```diff
@@ -242,6 +242,13 @@ var InternalSecureContext = class SecureContext {
         throw new TypeError("secureOptions argument must be an number");
       }
       this.secureOptions = secureOptions;
+
+      if (!$isUndefinedOrNull(options.privateKeyEngine) && typeof options.privateKeyEngine !== "string") {
+        throw $ERR_INVALID_ARG_TYPE("privateKeyEngine", "string", options.privateKeyEngine);
```

Comment:
if i'm reading node source correctly, should this be
```suggestion
        throw $ERR_INVALID_ARG_TYPE("privateKeyEngine", ["string", "null", "undefined"], options.privateKeyEngine);
```
?

Reply:
Just found that also — will update

Reply:
Fixed in 6f99c8950aef2202354813859a4d0febbd656e3d